CONTRIBUTING.md

```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
 
 (Run all of these commands from the project root)
 
-ruby 2.1.2 
+ruby 2.1.2
 
-bundle
+bundler
 
-  $ gem install bundle
-    
-node and npm 
+  $ gem install bundler
+
+node and npm (at this time `nodejs` version `0.10` is required. Install
+[nvm](https://github.com/creationix/nvm) then use it to install node and npm.)
+
+  $ nvm install 0.10
```

Comment:
Indeed this was an issue with the brew default nodejs. I inspected the [travis.yml](https://github.com/pivotal-cf/pivotal-ui/blob/master/.travis.yml) file to understand which version(s) of nodejs we're targeting. The project, via its CI configuration, is presently locked to 0.10. This documentation change makes that explicit.